src/components/Notification/BasicNotification.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useRef, useEffect } from 'react';
+import Stack from '@mui/material/Stack';
+import Snackbar from '@mui/material/Snackbar';
+import Alert from '@mui/material/Alert';
+import useNotification from '../../hooks/useNotification';
+
+const BasicNotification = ({ severity, message, id }) => {
+  const notify = useNotification();
```

Comment:
Similar with `addNotification`, you could destructure the `remove` function from `useNotification`.

Reply:
@leekahung when I try to destructure the remove function for the useNotification in basicNotification, it makes my test fail which confuses me because the function still works (still removes snackbar after 8 seconds) which means it must be importing. 

![Screen Shot 2023-08-09 at 3 19 11 PM](https://github.com/codeforpdx/PASS/assets/81979567/73379f05-5d1b-460e-862e-07b057f5a4c1)